windows.md

```diff
@@ -257,6 +257,9 @@ With those compatibility things out of the way, you're ready to start the system
     echo "export PGDATA=\"/c/Program Files/PostgreSQL/16/data\"" >> "$USERPROFILE/.bash_profile"
     echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> "$USERPROFILE/.bash_profile"
     source "$USERPROFILE/.bash_profile"
+    sed -i -e "s/^logging_collector = on/logging_collector = off/" \
```

Comment:
this `logging_collector` change is unrelated to this PR, which has the title "Add psql Language Settings for Linux and Windows"

Reply:
https://github.com/upleveled/system-setup/pull/80/commits/d49c0dc36c730091c6134bc198803c2f724d891a

Reply:
nice, you can move it to a new PR, so that we can review that separately

Reply:
nice, before we merge here, you can move it to a new PR, so that we can review that separately

Reply:
Done in https://github.com/upleveled/system-setup/pull/87